src/app/errors/routes.ts

```diff
@@ -1,16 +1,19 @@
 import Loadable from 'react-loadable';
 import { Route } from '../navigation/types';
+import { assertWindow } from '../utils';
 
 const CATCH_ALL = '/(.*)';
 
-export const notFound: Route<undefined> = {
+export const notFound: Route<undefined> & { getFullUrl: () => string, redirect: () => void } = {
   component: Loadable({
-    loader: () => import(/* webpackChunkName: "PageNotFound" */ './components/PageNotFound'),
+    loader: () => import(/* webpackChunkName: "LoaderCentered" */ './components/LoaderCentered'),
     loading: () => null,
-    modules: ['PageNotFound'],
-    webpack: /* istanbul ignore next */ () => [(require as any).resolveWeak('./components/PageNotFound')],
+    modules: ['LoaderCentered'],
+    webpack: /* istanbul ignore next */ () => [(require as any).resolveWeak('./components/LoaderCentered')],
   }),
-  getUrl: () => '/errors/404',
+  getFullUrl: () => 'https://openstax.org' + notFound.getUrl(),
+  getUrl: () => '/error/404',
   name: 'NotFound',
   paths: [CATCH_ALL],
+  redirect: () => assertWindow().location.replace(notFound.getFullUrl()),
```

Comment:
It has to redirect to the full url, because otherwise, puppeteer from `src/index.browserspec.tsx` will fail